sci-libs/scipy/scipy-1.3.3.recipe

```diff
@@ -0,0 +1,123 @@
+SUMMARY="Scientific algorithms library for Python"
+DESCRIPTION="SciPy is an open source library of scientific tools for Python. \
+SciPy supplements the popular numpy module, gathering a variety of high level \
+science and engineering modules together as a single package. SciPy includes \
+modules for graphics and plotting, optimization, integration, special \
+functions, signal and image processing, genetic algorithms, ODE solvers, and \
+others.
+
+This package does not contain ndimage module."
+HOMEPAGE="https://www.scipy.org/"
+COPYRIGHT=" 2001-2002 Enthought, Inc.
+	2003-2019 SciPy Developers"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/scipy/scipy/releases/download/v$portVersion/scipy-$portVersion.tar.xz"
+CHECKSUM_SHA256="6de202d30b9e9e0704972f2b9272b1ff1090e2b089ddbdccfcf6271ce5ffb2d9"
+SOURCE_DIR="scipy-$portVersion"
+PATCHES="scipy-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+# scipy can be build directly on x86
+# but I was unable to make recipe do
+# in the same way as "setarch x86" do
+SECONDARY_ARCHITECTURES="!x86"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cblas_reference$secondaryArchSuffix
+	lib:libarpack$secondaryArchSuffix
+	lib:libblis$secondaryArchSuffix
+	lib:liblapack$secondaryArchSuffix
+	lib:libopenblas$secondaryArchSuffix
+	lib:libumfpack$secondaryArchSuffix # not sure if actually used
+	numpy$secondaryArchSuffix
+	"
+
+PYTHON_PACKAGES=(python36 python3)
+PYTHON_VERSIONS=(3.6 3.7)
```

Comment:
Please remove 3.6.

Reply:
@korli should we start using 3.7, 3.8 and 3.9 in the recipes now?
On that note, maybe if we want 3.7 to be default we could use:
```diff
-PYTHON_PACKAGES=(python36 python3)
-PYTHON_VERSIONS=(3.6 3.7)
+PYTHON_PACKAGES=(python python38 python39)
+PYTHON_VERSIONS=(3.7 3.8 3.9)
```

Reply:
There is an open issue on python 3.8 or 3.9, would better fix it first